### DISCUSSION TITLE
Is downloading legal/allowed?

### DISCUSSION THREAD

**Discussion by KaptainAhoibrause** [KRTirtho/spotube#1914] [Q&A]
I've heard that Spotube in general is legal, because only the Spotify Metadata is used and the music comes (by default) from YouTube. But is downloading a song from YouTube legal?

**Comment by Peperehobbits01** [comment]
> I've heard that Spotube in general is legal, because only the Spotify Metadata is used and the music comes (by default) from YouTube. But is downloading a song from YouTube legal?

Well, from the YouTube guidelines if you don't have Premium no, they don't allow you to do that but will you get in trouble for it… No, not at all.

**Comment by KaptainAhoibrause** [comment]
OK, thanks!
